runtime-light/tl/tl-types.h

```diff
@@ -0,0 +1,25 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <cstdint>
+
+#include "runtime-core/runtime-core.h"
+#include "runtime-light/tl/tl-core.h"
+
+namespace tl {
+
+constexpr uint32_t K2_JOB_WORKER_RESPONSE_MAGIC = 0x3afb3a08;
+
+struct K2JobWorkerResponse final {
```

Comment:
Why K2JobWorkerResponse is a type, but K2InvokeJobWorker is a function?

Reply:
Do you mean why not both of them are TL-functions?

Reply:
I mean, that these entities have to have same type

Reply:
`K2InvokeJobWorker` is a function cause it's assumed that it returns a result of type `K2JobWorkerResponse`. In this PR we don't always return `K2JobWorkerResponse` actually cause we do have no reply job workers. We could also return actual `K2JobWorkerResponse` even for no reply job workers, but it's useless.

Reply:
okay, now it's clear